pkg/flags/flags.go

```diff
@@ -0,0 +1,47 @@
+package flags
+
+import (
+	"net/http"
+
+	ld "gopkg.in/launchdarkly/go-server-sdk.v4"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+)
+
+func flagMiddleware(next http.Handler, key string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		user := ld.NewUser(key)
+
+		ctx := appcontext.WithLDAppEnvUser(r.Context(), user)
+
+		next.ServeHTTP(w, r.WithContext(ctx))
+	})
+}
+
+// NewFlagMiddleware returns a handler with the flag middleware
+func NewFlagMiddleware(key string) func(http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		return flagMiddleware(next, key)
+	}
+}
+
+// FlagValues is the struct for our keys and values from LD
+type FlagValues map[string]interface{}
+
+// NewFetchFlags returns all flags from LD
+func NewFetchFlags() func(client ld.LDClient, user ld.User) FlagValues {
+	return func(c ld.LDClient, u ld.User) FlagValues {
+		return allFlags(c, u)
+	}
+
+}
+
+func allFlags(client ld.LDClient, user ld.User) FlagValues {
```

Comment:
Since for the immediate implementation we are only looking for flags-per-environment rather than targeting single users, why not just "hard code" an anonymous/single user here... This would remove the need to do any of the User placement on the Context.